telegram_bot.py

```diff
@@ -272,7 +285,10 @@ async def watch():
                 for chat_id in pilot_data.chat_ids:
                     log.debug(f"About to post {flight} to {chat_id=}")
                     try:
-                        await bot.send_message(chat_id, f"{flight.title}\n{flight.link}")
+                        flight_title = escape_md_v2(flight.title)
+                        flight_link = flight.link
+                        flight_username = extract_username(flight.link)
+                        await bot.send_message(chat_id, f"[{flight_title}]({flight_link}) \\[[{flight_username}](https://www.xcontest.org/world/en/pilots/detail:{flight_username})\\]", parse_mode="MarkdownV2")
```

Comment:
Can you use `flight.pilot.url` instead of `https://www.xcontest.org/world/en/pilots/detail:{flight_username})` please?